android/fastlane/metadata/android/de-DE/video.txt

```diff
@@ -0,0 +1 @@
+https://www.youtube.com/watch?v=1HkdjMGvsUk
```

Comment:
Is no video shown in different languages since the corresponding files are empty? Or is this used as fallback?

Reply:
I don't really know, I think it is not used as a fallback. This deployment information should match the state of the current deployment